### DISCUSSION TITLE
Remove Caddy?

### DISCUSSION THREAD

**Discussion by juststevestuff** [ubergeek77/Lemmy-Easy-Deploy#64] [General]
Any way to just take caddy out of this setup? I already use nginxproxymanager.  I know the FAQ says you can't help with the proxy configurations but hoping you might at least be able to explain how to remove caddy from the config. I've tried several times fiddling with the docker-compose.yml and templates but failing miserably.

**Comment by ubergeek77** [comment]
Sadly it won't be that easy. Caddy is an integral part as it does the routing for requests to the UI and the API. Read the Caddyfile to figure out what I mean.

You can maybe try to replicate that in Nginx, but I think you'd be better off disabling TLS/HTTPS using the options I provide and proxying to Caddy like that.

I haven't actually tried this since that's pretty hacky, so I won't have specific advice, sadly.

However, if you do investigate this and get it working, I will gladly take contributions to the FAQ to help other users in a similar situation!
